### PR TITLE
Add serviceability output to --time

### PR DESCRIPTION
**Pull request by nbrahms**
Let's inform users how to address timing issues.

PR checklist:
- [X] Documentation is up-to-date
- [X] Changelog is up-to-date
- [X] Change has no security implications (otherwise, ping security team)

### CODE REVIEW

File: semgrep/semgrep/formatter/text.py [hunk 1]
```diff
@@ -159,10 +159,10 @@ def _build_text_timing_output(
         total_matching_time = sum(i[1] for i in rule_timings.values())
 
         # Output information
-        yield f"\nSemgrep-core timing summary:"
```
Comment (nbrahms) [inline semgrep/semgrep/formatter/text.py:162]: I don't expect our users to know what "core" means

File: semgrep/semgrep/formatter/text.py [hunk 2]
```diff
@@ -171,14 +171,19 @@ def _build_text_timing_output(
...
+        yield f"\nSlowest {items_to_show} rules to run (excluding parse time)"
         slowest_rule_times = sorted(
             rule_timings.items(), key=lambda x: float(x[1][0]), reverse=True
         )[:items_to_show]
         for rule_id, (total_time, match_time) in slowest_rule_times:
             rule_id = truncate(rule_id, col_lim) + ":"
             yield f"{with_color('yellow', f'{rule_id:<71}')} run time {total_time:.4f}  match time {match_time:.4f}"
 
+        yield ""
+        yield f'{with_color("blue", "• To skip a file, run Semgrep with --exclude")}'
```
Comment (ievans) [inline semgrep/semgrep/formatter/text.py:183]: Maybe link to https://semgrep.dev/docs/ignoring-findings/#file-paths instead?

File: semgrep/semgrep/formatter/text.py [hunk 3]
```diff
@@ -171,14 +171,19 @@ def _build_text_timing_output(
...
+        yield f'{with_color("blue", "• To skip a rule, remove it from your config")}'
```
Comment (ievans) [inline semgrep/semgrep/formatter/text.py:184]: Can we link to the dashboard?

**Comment by brendongo** [comment]
Can we add a quick test for the output of this

**Pull request closed without merging**